libs/media-player/src/lib/default-play-pause-button.tsx

```diff
@@ -0,0 +1,25 @@
+import VolumeUpIcon from '@mui/icons-material/VolumeUp';
+import { IconButton } from '@mui/material';
+
+// interface PlayPauseIconProps {
+//     isPlaying: boolean;
+// }
+
+export interface MediaPlayerButtonProps {
+    isDisabled: boolean;
+    onButtonClick: () => void;
+}
+
+const PlayPauseIcon = () => {
```

Comment:
Call this  PlayIcon. But is it really needed? Can't you just inline the volume up icon below?